app/core/layouts/Header.tsx

```diff
@@ -27,11 +28,13 @@ const StyledHeaderButton = styled(Button)(({ theme }) => ({
   },
 }));
 
-const Header = ({ existApplicant }: IProps) => {
+
+const Header = ({ existApplicant, applicantId }: IProps) => {
   const currentUser = useOptionalUser();
```

Comment:
I wonder if we could add a useOptionalApplicant() function, to not have to pass the `applicantId` as a parameter... 🤔